src/content/perf/shader.md

```diff
@@ -69,29 +40,39 @@ The following screenshot shows an example timeline tracing.
 
 ![A tracing screenshot verifying jank](/assets/images/docs/perf/render/tracing.png){:width="100%"}
 
-## What do we mean by "first run"?
+## How to reduce compilation jank
+
+If you're not using Flutter's Impeller graphics renderer and you're creating an
+iOS app, you can minimize shader compilation jank by bundling precompiled
+shaders with your iOS app. To learn more, see the iOS app development
+documentation.
 
-On iOS, "first run" means that the user might see
-jank when an animation first occurs every time
-the user opens the app from scratch.
+If you're not using Flutter's Impeller graphics renderer and you're creating an
```

Comment:
We don't actually recommend/support using the cached sksl on android - we found that the shaders generated weren't compatible across different devices so its not safe to do. I don't know where we documented this

Reply:
I would vote for removing all documentation on SkSL warmup, as we plan to remove all of the tooling for it in the next quarter or so.

Reply:
I was wondering about this. I'll remove the documentation since it appears to be obsolete.